TypeCobol/Compiler/Parser/CodeElementBuilder/CobolCodeElementBuilder.cs

```diff
@@ -2305,5 +2305,12 @@ public override void EnterSavepointStatement([NotNull] CodeElementsParser.Savepo
             CodeElement = savepointStatement;
             SavepointStatementChecker.OnCodeElement(savepointStatement, context);
         }
+
+        public override void EnterExecuteImmediateStatement(
+            [NotNull] CodeElementsParser.ExecuteImmediateStatementContext context)
+        {
+            Context = context;
+            CodeElement = _sqlCodeElementBuilder.CreateExecuteImmediateStatement(context);
```

Comment:
I think we can add a simple checker to check that the hostvariable does not have any indicator variable